mtags/src/main/scala/scala/meta/internal/metals/docstrings/MarkdownGenerator.scala

```diff
@@ -36,7 +36,23 @@ object MarkdownGenerator {
       .mkString
   }
 
-  def toMarkdown(c: Comment): String = {
+  def toMarkdown(c: Comment, docstring: String): String = {
+    def sortInSection(
```

Comment:
I did not mean using Scala lang syntax parser but a simple hand written parser for what you need to extract. Honestly I think it may be enough to always collect the two words after each ` @`. But sure, you don't have to do this. This should never be a very long string.

Reply:
We should be fine since parsing scaladoc is lazy. We could make a regex instead and just get the order in one go, but this should also be fine.